plugins/templates/labhub/hello-world.jinja2.md

```diff
@@ -0,0 +1,11 @@
+Welcome @{{ target }}! :tada:
+
+A developer or maintainer will soon invite you to the orgazinzation.
```

Comment:
`will soon invite you` --> `will invite you to the organization soon`

Reply:
I guess both are correct ways to write this statement?